lib/roo_on_rails/railties/http.rb

```diff
@@ -34,7 +34,7 @@ class HTTP < Rails::Railtie
           end
 
           # Don't use SslEnforcer in test environment as it breaks Capybara
-          unless Rails.env.test?
+          unless Rails.env.test? || ENV.fetch('ROO_ON_RAILS_DISABLE_SSL_ENFORCEMENT', 'YES').to_s =~ /\A(YES|TRUE|ON|1)\Z/i
```

Comment:
I really don't have any context, but I'd worry that setting this var to `YES` by default could be a bad idea—any downstream users of this gem would have SSL turned off, maybe unexpectedly.

Reply:
D'oh – you're totally right – that default `YES` is in there erroneously. Now fixed.